src/components/MediaCard/MediaCard.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useEffect } from 'react';
+import CardMedia from '@mui/material/CardMedia';
+import Typography from '@mui/material/Typography';
+
+import { formatLargeNumber } from '../../utils/numbers';
+import { VIDEO_HEIGHT } from '../../constants';
+import { Stats } from '../../types/userFeedTypes';
+import fallbackSrc from '../../assets/Tik-tok-logo.jpeg';
+import { StyledCard, StyledDiv, StyledIcon } from './styles';
+
+type Props = {
+  id: string
+  videoURL: string
+  error: string | null
+  setError: (a: string | null) => void
+  stats?: Stats
+}
+
+const MediaCard = ({ id, videoURL, error, setError, stats }: Props) => {
```

Comment:
These props "error" and "setError"  is looking like deep prop drilling to the component. Required additional time to check, what is exactly this error and setError,  which component they coming from. This reduces readability.

Reply:
I don’t think that props drilling is always a bad thing, especially if the app is small and you use it only a couple of times. For this particular app, in my personal opinion, using any state management tool would be an overkill.